### PR TITLE
Update installation guide

### PR DESCRIPTION
**Pull request by kitchoi**
I have added the mention of Qt in the installation guide, added the use of pip, updated links and ETS installation instructions (which I am tempted to remove and leave it the user to follow the link).
I have also updated the requirements and added notes on the latest versions of VTK.

Notes:
1.  I have not checked the Windows and Mac OS Snow Leopard sections
2. "numpy > 1.1.1" is taken from PyPI page of mayavi, I have not tested it myself

Related to #290 
@prabhuramachandran, @jdmarch can you take a look please?

### CODE REVIEW

File: docs/source/mayavi/installation.rst [hunk 1]
```diff
@@ -43,29 +43,40 @@ Installing ready-made distributions
...
+Doing it yourself
+------------------
+
+.. _requirements-for-install:
+
 Requirements for manual installs
---------------------------------
+................................
 
 If you are not using full, ready-made, scientific Python distribution,
```
Comment (prabhuramachandran) [inline docs/source/mayavi/installation.rst]: "If you are not using a full" an "a" is missing.

**Comment by prabhuramachandran** [comment]
LGTM, thanks!  Will wait for @jdmarch to comment.

File: docs/source/mayavi/installation.rst (hunk 1 above)
Comment (jdmarch) [inline docs/source/mayavi/installation.rst]: s/full/a full/

File: docs/source/mayavi/installation.rst [hunk 2]
```diff
@@ -43,29 +43,40 @@ Installing ready-made distributions
...
+   newer than 6.2.0, please check the Changelog_ for the current status
+   of Mayavi.
+
+.. _Changelog: auto/changes.html
 
 The following requirements are really optional but strongly recommended,
 especially if you are new to Mayavi:
 
-    * wxPython_ 2.8.x
+    * PyQt4_ 4.8.x or wxPython_ 2.8.x
```
Comment (jdmarch) [inline docs/source/mayavi/installation.rst]: What about PySide?

**Comment by jdmarch** [comment]
(removed my braindead rejection of "traitsui_" reference)

File: docs/source/mayavi/installation.rst [hunk 3]
```diff
@@ -295,78 +326,39 @@ The bleeding edge: Git
...
+  cd ets
+  python setup.py develop
+  cd ..
+  ets -h
 
-       The `ets.py` downloads the entire ETS, which is more than you need
-       to build Mayavi. As the extra packages have additional
-       dependencies, they may render the build harder. You can remove
-       safely the following directories::
+To clone all the ETS projects::
```
Comment (jdmarch) [inline docs/source/mayavi/installation.rst]: @kitchoi I agree with your qualm / temptation -- suggest replacing this with a reference to https://github.com/enthought/ets

**Comment by jdmarch** [comment]
@kitchoi On quick review, LGTM, with small notes.

**Comment by kitchoi** [comment]
Thanks @jdmarch !  I have updated the doc per your suggestions.

**Comment by prabhuramachandran** [comment]
LGTM. Thanks.

**Pull request merged**